test/blackbox-tests/test-cases/github4345.t

```diff
@@ -0,0 +1,11 @@
+This is a reproduction case from issue #4345
```

Comment:
Could you add a brief description here anyway? It's a bit confusing to understand this test because there's no output from any of the commands.